NorthstarDLL/mods/autodownload/moddownloader.h

```diff
@@ -0,0 +1,152 @@
+class ModDownloader
+{
+  private:
+	const char* STORE_URL = "https://gcdn.thunderstore.io/live/repository/packages/";
+	const char* MODS_LIST_URL = "https://raw.githubusercontent.com/R2Northstar/VerifiedMods/master/mods.json";
```

Comment:
Perhaps this should be a convar so that people can configure it? Not sure if worth doing

Reply:
I kinda see the point of doing this, but as convars are modifiable by mods, that would create a security issue :/

Reply:
use a command line arg then.

Reply:
Good point!
Addressed in 8a1792c5c19e8b206f5e77b40a8b5df49afa2753